examples/framesjs-starter/app/examples/simplified-api/[[...routes]]/route.tsx

```diff
@@ -0,0 +1,72 @@
+import { AnyJson } from "../new-api";
+import {
+  FrameImage,
+  Frame,
+  FramesApp,
+  FrameLink,
+  renderFramesToResponse,
+} from "../react-api";
+
+type FrameState = {
+  clicked: number;
+};
+
+function isFrameState(
+  frameState: AnyJson | undefined
+): frameState is FrameState {
+  if (frameState && typeof frameState === "object" && "clcked" in frameState) {
+    return typeof frameState.clicked === "number";
+  }
+
+  return false;
+}
+
+export const framesApp = (
+  <FramesApp>
+    <Frame index>
```

Comment:
I think `path="/"` here is okay. It's not immediately clear what the index prop is doing

Reply:
Makes sense but also `index` is used to detect initial frame. So perhaps `initial` prop would be better?